src/groups/mwc/mwcst/mwcst_statvalue.h

```diff
@@ -339,7 +339,7 @@ class StatValue {
     /// Return the snapshot referred to by the specified `location`.  The
     /// behavior is undefined unless
     /// `location.level() < numLevels()` and
-    /// `location.index() <= historySize(location.level())`
+    /// `location.index() < historySize(location.level())`
```

Comment:
The actual condition is more strict:
https://github.com/bloomberg/blazingmq/blob/1387dfab4bc21f1df279ff84c617b0089e90fd68/src/groups/mwc/mwcst/mwcst_statvalue.h#L700